bpmn-statistic/src/com/axonivy/utils/bpmnstatistic/internal/ProcessUtils.java

```diff
@@ -175,6 +176,9 @@ public static List<IWebStartable> getAllProcesses() {
   public static Map<String, List<IProcessWebStartable>> getProcessesWithPmv() {
     Map<String, List<IProcessWebStartable>> result = new HashMap<>();
     for (IWebStartable process : getAllProcesses()) {
+      if (process instanceof CaseMapWebStartable) {
```

Comment:
please handle this logic in method getAllProcesses() ( maybe we need to edit logic of (ProcessUtils::isIWebStartableNeedToRecordStatistic)